train/uft/config.yml

```diff
@@ -1,13 +1,5 @@
-denoiser_adam_lr: 0.0001
-discriminator_adam_lr: 0.0001
-num_epochs: 10
-step_lr_step_size: 30
-step_lr_gamma: 0.1
-batch_size: 4
-print_loss_interval: 100
-calc_eval_loss_interval: 100
-denoiser_loss_b1: 0.5
-denoiser_loss_b2: 0.5
-checkpoint_path: C:\Users\super\Documents\1.School\FYP\fyp_project\src\toenet\checkpoint\checkpoint.pth.tar
-images_dir: C:\Users\super\Documents\1.School\FYP\fyp_project\Data\input\unpaired
-save_dir: C:\Users\super\Documents\1.School\FYP\fyp_project\Data\output
```

Comment:
why are these parameters removed?

Reply:
all parameters are initialised in the tuning.py in the get_params(trial) function to be called in the optuna objective function

Reply:
![image](https://github.com/superleesa/final_year_project/assets/96767995/0a461a8d-9261-4855-9c9c-1b59f9e484a4)

Reply:
the directories only would be in the config.yml

Reply:
i see but the original train.py for UFT uses those configurations and so deleting them cause the train.py to no longer work. can you can create a separate config file for the tuning pls?

Reply:
@weeenieee

Reply:
Ok sure, can